### DISCUSSION TITLE
Getting a unicorn error with HTML meta redirect on homepage

### DISCUSSION THREAD

**Discussion by jack126guy** [community/community#23471] [Other Feature Feedback, Questions, & Ideas]
<p>My homepage <a href="https://jack126guy.github.io/" rel="noopener nofollow ugc">https://jack126guy.github.io/</a> doesn’t seem to be working. Every time I try to access it I get the “unicorn” error page.</p>
<p>I thought this might have to do with the <a href="https://www.githubstatus.com/incidents/zhtplv7zd052" rel="noopener nofollow ugc">recent incident</a> because I made a change while it was ongoing. However, I tried pushing another change after the incident was supposed to be resolved, and I’m still getting the same error page.</p>
<p>I also wonder if this has to do with the fact that the page has an HTML meta redirect, but I have read that others are able to do this with no problem.</p>
<p>Note: I also have other sites for specific repositories (such as <a href="https://jack126guy.github.io/magic28/" class="inline-onebox" rel="noopener nofollow ugc">Magic28</a> ) which work fine.</p>

**Comment by jack126guy** [comment, marked as answer]
<p>Right after I submitted the original post it started working :neutral_face:</p>

**Comment by jfullstackdev** [comment]
<p>yup it’s working</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/5/9507c8ebf7bf068c2dc58f9b18f6a3930a3f11c0.png" data-download-href="/uploads/short-url/lgnOY0tCQ54LD8yh6OwHGzDiYhO.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077260-046800ec-2b2b-4f15-8781-63f0b72c135f.png" alt="image" data-base62-sha1="lgnOY0tCQ54LD8yh6OwHGzDiYhO" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/5/9507c8ebf7bf068c2dc58f9b18f6a3930a3f11c0_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/5/9507c8ebf7bf068c2dc58f9b18f6a3930a3f11c0_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/5/9507c8ebf7bf068c2dc58f9b18f6a3930a3f11c0_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/5/9507c8ebf7bf068c2dc58f9b18f6a3930a3f11c0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 34 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>it will take some time to be deployed</p>

**Comment by jack126guy** [comment]
<p>It stopped working again for some reason. I tried to rerun the Action but that didn’t seem to do anything. I also enabled HTTPS to see if that might help.</p>
